sdk-reference/mobile/react-native.mdx

```diff
@@ -15,6 +15,74 @@ First, install the Rownd SDK for React Native.
 npm install @rownd/react-native
 ```
 
+### Expo development
+
+1. Add `@rownd/native` as a plugin to your `app.json` file.
```

Comment:
This should be `@rownd/react-native`, right?

Reply:
Wow just seeing these comments. Updated